DAQ/src/ArtBinaryPacketsFromDigis_module.cc

```diff
@@ -1096,23 +1097,13 @@ void ArtBinaryPacketsFromDigis::processCrvData(art::Event& evt, uint64_t& eventN
 //--------------------------------------------------------------------------------
 // crate a crvPacket from the digi
 //--------------------------------------------------------------------------------
-uint8_t ArtBinaryPacketsFromDigis::compressCrvDigi(int adc) {
+int16_t ArtBinaryPacketsFromDigis::compressCrvDigi(int16_t adc) 
+{
```

Comment:
Is there a reason this is int16_t and not uint16_t?  Can the ADC really take on negative values?  If so, what does that mean?  Will downstream code produce a correct interpretation if it is negative?  If not, I suggest changing to uint16_t.

Reply:
Yes, the current firmware can (and does) return negative ADC values. That's why I had to change it to the signed int. The downstream code has no problem with negative ADC values; it only cares about the difference to the pedestal (which can also be negative).